### ISSUE TITLE
Training Code

### ISSUE THREAD

**Issue by ayd-ocan** [bytedance/tarsier#13]
Thanks for your impressive work ! Are you planning to release training code soon?

**Comment by princepride** [comment]
I have successfully written the training code. First, you need to know how to perform batch inference. Then, based on the intermediate computation results from the inference, find a way to construct attention_mask, input_ids, pixel_values, and labels. After that, you can proceed with the standard training steps.

**Comment by orzgugu** [comment]
> I have successfully written the training code. First, you need to know how to perform batch inference. Then, based on the intermediate computation results from the inference, find a way to construct attention_mask, input_ids, pixel_values, and labels. After that, you can proceed with the standard training steps.
Could you please provide your SFT code? Thank you very much for your work

**Comment by linzhiqiu** [comment]
Is there no plan on releasing the training/finetuning code?
